Sources/FluentMySQL/MySQLDatabase.swift

```diff
@@ -33,6 +33,20 @@ public final class MySQLDatabase: Service {
         self.password = password
         self.database = database
     }
+    
+    /// Initialize MySQLDatabase with a DB URL
+    public convenience init?(databaseURL: String) {
+        guard let url = URL(string: databaseURL),
+            url.scheme == "mysql",
+            url.pathComponents.count >= 2,
```

Comment:
What happens to path components after index 1, should this be `==` instead of `>=`?

Reply:
You're right. There should only ever be two path components
`mysql://[host][,failoverhost...][:port]/[database][?propertyName1][=propertyValue1][&propertyName2][=propertyValue2]... `